tox.ini

```diff
@@ -1,8 +1,11 @@
 [tox]
 envlist =
+    py36-{docs,flake8},
     py{27,33,34,py,py3}-django18-{pg,sqlite},
-    py{27,34,35,py}-djangotrunk-{pg,sqlite},
-    py{27,py}-django{18,trunk}-mysql,
+    py{27,34,35,py}-django{19,110}-{pg,sqlite},
+    py{27,34,35,36,py}-django111-{pg,sqlite},
+    py{35,36}-djangotrunk-{pg,sqlite},
+    py{27,py}-django{18,19,110,111}-mysql,
```

Comment:
@brad 👍  I'm glad we can continue supporting 1.8 and 1.9 without any special cases.